### DISCUSSION TITLE
Safe upgrade from old version

### DISCUSSION THREAD

**Discussion by elect86** [bisq-network/bisq#5657] [Q&A] [answered]
Hi,

can I safely upgrade from 0.8.0 to the current version without risking to lose all my history?

Or better to upgrade version by version till the current one?

**Comment by elect86** [comment, marked as answer]
Solved by replacing the application folder `~/.local/share/Bisq`

**Reply by ripcurlx** [reply]
Doing such a big version jump it is always a good idea to do a backup first. Just out of interest: Why did you came back after such a long time? 😄  Welcome back!

**Reply by elect86** [reply]
I did indeed.

Long and boring story, but I picked up interest again :)
